### PR TITLE
clean up the header / hamburger

### PR DESCRIPTION
**Pull request by meyer-mcmains**
After some trial and error I got this to work without interfering with the actual page :rocket:

I also cleaned up the styles a bit (I think there was some leftover stuff from experimenting with the design) :crayon:

### CODE REVIEW

**Review by BenBrownCreative — APPROVED** [review]
this is so much simpler than my crazy solution

File: client/src/components/Header/index.js [hunk 1]
```diff
@@ -22,32 +18,33 @@ const Wrapper = styled.div`
   }
 `;
 
+const HeaderBar = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem 2rem;
```
Comment (BenBrownCreative) [inline client/src/components/Header/index.js:26]:
```suggestion
  padding: .5rem 2rem;
```

Comment (BenBrownCreative) [inline client/src/components/Header/index.js:26]:
```suggestion
  padding: .5rem 2rem;
```

File: client/src/components/Header/index.js [hunk 2]
```diff
@@ -22,32 +18,33 @@ const Wrapper = styled.div`
...
-      <span ref={wrapperRef}>
+    <>
+      <HeaderBar>
+        <div>logo</div>
         <Hamburger isOpen={isOpen} handleToggle={toggleIsOpen} />
-        <Menu isOpen={isOpen} handleToggle={toggleIsOpen} />
-      </span>
-    </Wrapper>
+      </HeaderBar>
+      <Menu isOpen={isOpen} handleToggle={toggleIsOpen} />
```
Comment (BenBrownCreative) [inline client/src/components/Header/index.js:45]:
```suggestion
          <Menu isOpen={isOpen} handleToggle={toggleIsOpen} />
      </HeaderBar>
      
```

File: client/src/components/Header/index.js [hunk 3]
```diff
@@ -22,32 +18,33 @@ const Wrapper = styled.div`
...
+const HeaderBar = styled.div`
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1.5rem 2rem;
+  position: fixed;
+  z-index: 1000;
+  top: 0;
+`;
```
Comment (BenBrownCreative) [inline client/src/components/Header/index.js:30]:
```suggestion
  background: ${p => p.theme.color.darkBlue};
`;
```

File: client/src/components/Hamburger/index.js [hunk 4]
```diff
@@ -2,24 +2,9 @@ import React from 'react';
...
-  border: 0;
-  margin: 0;
-  overflow: visible;
-  outline: none;
-  position: absolute;
   padding: 1rem;
-  top: 1rem;
-  right: 0.5rem;
-  z-index: 111;
 `;
```
Comment (BenBrownCreative) [inline client/src/components/Hamburger/index.js:8]:
```suggestion
z-index: 11;
`;
```

**Review by BenBrownCreative — CHANGES_REQUESTED** [review]
move the menu inside the header wrapper and give the hamburger a higher z-index. this will position the menu behind the hamburger and in front of the header background.

**Comment by meyer-mcmains** [comment]
@BenBrownCreative I think this is good to go now.

**Pull request merged**